src/Coordination/KeeperServer.cpp

```diff
@@ -96,23 +104,67 @@ KeeperServer::KeeperServer(
     : server_id(configuration_and_settings_->server_id)
     , coordination_settings(configuration_and_settings_->coordination_settings)
     , state_machine(nuraft::cs_new<KeeperStateMachine>(
-                        responses_queue_, snapshots_queue_,
-                        configuration_and_settings_->snapshot_storage_path,
-                        coordination_settings,
-                        checkAndGetSuperdigest(configuration_and_settings_->super_digest)))
-    , state_manager(nuraft::cs_new<KeeperStateManager>(server_id, "keeper_server", configuration_and_settings_->log_storage_path, config, coordination_settings))
+          responses_queue_,
+          snapshots_queue_,
+          configuration_and_settings_->snapshot_storage_path,
+          coordination_settings,
+          checkAndGetSuperdigest(configuration_and_settings_->super_digest)))
+    , state_manager(nuraft::cs_new<KeeperStateManager>(
+          server_id, "keeper_server", configuration_and_settings_->log_storage_path, config, coordination_settings))
     , log(&Poco::Logger::get("KeeperServer"))
+    , is_recovering(config.has("keeper_server.force_recovery") && config.getBool("keeper_server.force_recovery"))
 {
     if (coordination_settings->quorum_reads)
         LOG_WARNING(log, "Quorum reads enabled, Keeper will work slower.");
 }
 
-void KeeperServer::startup(bool enable_ipv6)
+struct KeeperServer::KeeperRaftServer : public nuraft::raft_server
```

Comment:
Need a small comment like `Tiny wrapper on top of nuraft::raft_server which add ...`